jobs/crontab

```diff
@@ -2,3 +2,4 @@
 
 11 * * * * /root/update_top_trending.sh
 * * * * 1 /root/MetaMaint.sh
+47 3 * * * /root/update_spam_blacklist.sh
```

Comment:
Any specific reason for this time? If there is none, I think we should change it to something less specific, `00:01` or `01:00` sound like good times to me.